thin-backend-react/auth-ui.tsx

```diff
@@ -124,7 +127,7 @@ function LoginForm() {
         <div>
             <button type="submit" disabled={isLoading}>
                 {isLoading
-                    ? <span className="thin-login-spinner-border thin-login-spinner-border-sm" role="status" aria-hidden="true"></span>
+                    ? <LoadingSpinner small/>
```

Comment:
consider allowing the spinner to be replaced via props, as apps will often have their own style of spinner.